js/ui/ui.css

```diff
@@ -93,6 +93,18 @@
   font-size: 0.8em;
 }
 
+.respec-gh-label {
```

Comment:
Just realized that this CSS is in the wrong place.  It should be moved to:

```
js/core/css/issues-notes.css
```

The "js/ui/ui.css" is only for the ReSpec pill, and is removed from the document on save.

Reply:
To get these to show up:

```CSS
.issue a.respec-gh-label {
  padding: 5px;
  margin: 0 2px 0 2px;
  font-size: 10px;
  text-transform: none;
  text-decoration: none;
  font-weight: bold;
  border-radius: 4px;
  position: relative;
  bottom: 2px;
}

.issue a.respec-label-dark {
  color: #fff;
  background-color: #000;
}

.issue a.respec-label-light {
  color: #000;
  background-color: #fff;
}

```

Reply:
Yeah sure